hack/functest.sh

```diff
@@ -12,6 +12,7 @@ GOBIN="${GOBIN:-$GOPATH/bin}"
 GINKGO=$GOBIN/ginkgo
 
 if ! [ -x "$GINKGO" ]; then
+	export GOFLAGS=""
```

Comment:
Could you also leave a comment, something like: 
# The golang-1.13 image used in OpenShift CI enforces vendoring and go get is disabled. Workaround that by unsetting it.

Reply:
Done